pkg/util/kubernetes/kubelet/kubelet_common.go

```diff
@@ -82,6 +82,12 @@ func KubeContainerIDToTaggerEntityID(ctrID string) (string, error) {
 	return "", fmt.Errorf("can't extract an entity ID from container ID %s", ctrID)
 }
 
+// RawContainerIDToTaggerEntityID builds an entity ID from a raw container ID
+// (i. e. not including the <runtime>:// prefix).
+func RawContainerIDToTaggerEntityID(ctrID string) string {
+	return containers.ContainerEntityName + ctrID
+}
+
```

Comment:
We already have this function https://github.com/DataDog/datadog-agent/blob/main/pkg/util/containers/entity.go#L31